### DISCUSSION TITLE
link to the subreddit from the main nicegui.io page?

### DISCUSSION THREAD

**Discussion by metaperl** [zauberzeug/nicegui#919] [Ideas / Feature Requests]
[The NiceGUI homepage](https://nicegui.io) links to discord and github. Shouldnt it also link to the subreddit?

**Comment by falkoschindler** [comment]
Good idea, @metaperl! I just added the link in [21e197a](https://github.com/zauberzeug/nicegui/commit/21e197ab003a4e97c2ae4d1d37397501f225e097). Will be online with the next release.
